daisy/step.go

```diff
@@ -69,8 +69,12 @@ type Step struct {
 	testType stepImpl
 }
 
-// NewStep creates a Step with given name and timeout  with the specified workflow
+// NewStep creates a Step with given name and timeout  with the specified workflow.
```

Comment:
I would create a new method (NewStepWithDefaultTimeout?) for this purpose because timeout<=0 looks like a hidden logic

Reply:
+1